src/Tools/CoreFx.Tools/UpdateVSConfigurations.cs

```diff
@@ -276,7 +276,8 @@ private void UpdateSolution(ITaskItem solutionRootItem)
                 foreach (var slnProject in projectFolder.Projects)
                 {
                     string projectName = Path.GetFileNameWithoutExtension(slnProject.ProjectPath);
-                    string relativePathFromCurrentDirectory = slnProject.ProjectPath.Replace(solutionRootPath, "");
+                    // Normalize the directory separators to the windows version given these are projects for VS and only work on windows.
+                    string relativePathFromCurrentDirectory = slnProject.ProjectPath.Replace(solutionRootPath, "").Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
```

Comment:
I don't think this works. You need to forcibly replace `/` with `\`. On Windows, these paths should already have the correct separators, and outside Windows, this doesn't make sense because `DirectorySeparatorChar` and `AltDirectorySeparatorChar` are both `/` (back-slash is not a valid separator).